LICENSE.md

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 2014-2017 EPFL
```

Comment:
This is not needed, I propose "2017 - Scalameta maintainers"

Reply:
:+1: